components/brave_vpn/resources/panel/components/status-indicator/index.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import * as S from './style'
+import locale from '../../constants/locale'
+interface Props {
+  isConnected: boolean
+}
+
+function StatusIndicator (props: Props) {
+  return (
+    <S.Box>
+      <S.Indicator isActive={props.isConnected} />
+      <S.Text>
+        {props.isConnected
+          ? locale.connectedText
```

Comment:
I think it's fine because panel is still in-progress.
In the future, we should get translated text like this - https://github.com/brave/brave-core/blob/master/components/brave_new_tab_ui/widgets/ftx/components/summary.tsx#L72